src/client/apps/edit/components/header/yoast/index.tsx

```diff
@@ -0,0 +1,249 @@
+import { Box, color, Flex, space } from "@artsy/palette"
+import { avantgarde } from "@artsy/reaction/dist/Assets/Fonts"
+import Icon from "@artsy/reaction/dist/Components/Icon"
+import {
+  Input,
+  StyledInput,
+  Title,
+} from "@artsy/reaction/dist/Components/Input"
+import { ArticleData } from "@artsy/reaction/dist/Components/Publishing/Typings"
+import { StaticCollapse } from "@artsy/reaction/dist/Components/StaticCollapse"
+import React, { Component } from "react"
+import { connect } from "react-redux"
+import styled from "styled-components"
+import {
+  App as YoastApp,
+  SnippetPreview as YoastSnippetPreview,
+} from "yoastseo"
+import { setYoastKeyword } from "../../../../../actions/edit/editActions.js"
```

Comment:
i'd say go ahead and convert that file to a `.ts` 
also @eessex don't we have module aliases setup in positron? (e.g. `edit/actions/edit/editActions`)